packages/input/src/index.ts

```diff
@@ -1,9 +1,33 @@
-import { FrameworkConfiguration, PLATFORM } from 'aurelia-framework';
+import { FrameworkConfiguration, PLATFORM, ValueAttributeObserver, bindingMode } from 'aurelia-framework';
+import { AureliaUX } from '@aurelia-ux/core';
 
 export { UxInputTheme } from './ux-input-theme';
 
 export function configure(config: FrameworkConfiguration) {
+  config.container.get(AureliaUX).registerUxElementConfig(uxInputConfig);
   config.globalResources([
     PLATFORM.moduleName('@aurelia-ux/input/ux-input')
   ]);
 }
+
+const uxInputConfig = {
```

Comment:
We need to be sure to extend this configuration to the [`@aurelia-ux/components`](https://github.com/aurelia/ux/tree/master/packages/components) package.